Publishing images is failing since updating the CI to Ubuntu 22.04

<!-- **Are you in the right place?**
1. For issues or feature requests, please create an issue in this repository.
2. For general technical and non-technical questions, we are happy to help you on our [Rook.io Slack](https://slack.rook.io/).
3. Did you already search the existing open issues for anything similar? -->

**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**
The publish images CI is failing in master and 1.14 branches since #14166 was merged to update the CI to Ubuntu 22.04. 
See the [history of the CI failures](https://github.com/rook/rook/actions/workflows/push-build.yaml).

The logs of a [failed job](https://github.com/rook/rook/actions/runs/8987755608/job/24687015677) show:
```
=== installing manifest-tool
mkdir -p /home/runner/work/rook/rook/.cache/tools/linux_amd64
curl -sL https://github.com/estesp/manifest-tool/releases/download/v1.0.2/manifest-tool-linux-amd64 > /home/runner/work/rook/rook/.cache/tools/linux_amd64/manifest-tool-v1.0.2
chmod +x /home/runner/work/rook/rook/.cache/tools/linux_amd64/manifest-tool-v1.0.2
      Successfully remade target file '/home/runner/work/rook/rook/.cache/tools/linux_amd64/manifest-tool-v1.0.2'.
     Finished prerequisites of target file 'publish.manifest.image.ceph'.
    Must remake target 'publish.manifest.image.ceph'.
/home/runner/work/rook/rook/.cache/tools/linux_amd64/manifest-tool-v1.0.2 push from-args --platforms linux/amd64 linux/arm64, --template rook/ceph-ARCH:v1.14.0.115.g517b1bf12 --target rook/ceph:v1.14.0.115.g517b1bf12
time="2024-05-07T15:14:02Z" level=fatal msg="You must specify all three arguments --platforms, --template and --target"
make: *** [Makefile:192: publish.manifest.image.ceph] Error 1
make: Leaving directory '/home/runner/work/rook/rook/build/release'
Error: Process completed with exit code 2.
```

Perhaps the argument is getting confused, since the "ARCH" param is not getting evaluated in that command:
```
--template rook/ceph-ARCH:v1.14.0.115.g517b1bf12
```

**Expected behavior:**
Publish images CI should pass

**How to reproduce it (minimal and precise):**
<!-- Please let us know any circumstances for reproduction of your bug. -->
Merge to master or 1.14 branches and see the failure.

--- a/build/release/Makefile
+++ b/build/release/Makefile
@@ -70,6 +70,7 @@ DOCKER_REGISTRY ?= rook
 REGISTRIES ?= $(DOCKER_REGISTRY)
 IMAGE_ARCHS := $(subst linux_,,$(filter linux_%,$(PLATFORMS)))
 IMAGE_PLATFORMS := $(subst _,/,$(subst $(SPACE),$(COMMA),$(filter linux_%,$(PLATFORMS))))
+IMAGE_PLATFORMS_COMMA=$(shell echo "$(IMAGE_PLATFORMS)" | sed 's/ /,/')
 
 S3_BUCKET ?= rook.releases
 S3_CP := aws s3 cp --only-show-errors
@@ -189,11 +190,11 @@ endef
 $(foreach r,$(REGISTRIES), $(foreach i,$(IMAGES), $(foreach a,$(IMAGE_ARCHS),$(eval $(call repo.targets,$(r),$(i),$(a))))))
 
 publish.manifest.image.%: publish.all.images $(MANIFEST_TOOL)
-	$(MANIFEST_TOOL) push from-args --platforms $(IMAGE_PLATFORMS) --template $(DOCKER_REGISTRY)/$*-ARCH:$(VERSION) --target $(DOCKER_REGISTRY)/$*:$(VERSION)
+	$(MANIFEST_TOOL) push from-args --platforms $(IMAGE_PLATFORMS_COMMA) --template $(DOCKER_REGISTRY)/$*-ARCH:$(VERSION) --target $(DOCKER_REGISTRY)/$*:$(VERSION)
 
 # add the "master" tag to the master image, but do not add the "release" tag for the release channel
 promote.manifest.image.%: promote.all.images $(MANIFEST_TOOL)
-	[ "$(CHANNEL)" = "release" ] || $(MANIFEST_TOOL) push from-args --platforms $(IMAGE_PLATFORMS) --template $(DOCKER_REGISTRY)/$*-ARCH:$(VERSION) --target $(DOCKER_REGISTRY)/$*:$(CHANNEL)
+	[ "$(CHANNEL)" = "release" ] || $(MANIFEST_TOOL) push from-args --platforms $(IMAGE_PLATFORMS_COMMA) --template $(DOCKER_REGISTRY)/$*-ARCH:$(VERSION) --target $(DOCKER_REGISTRY)/$*:$(CHANNEL)
 
 build.images: build.all.images
 publish.images: $(addprefix publish.manifest.image.,$(IMAGES))

